app/components/edit/collapse_item_component.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Edit
+  # Component for displaying a collapsible item on the submitter form.
+  class CollapseItemComponent < ApplicationComponent
```

Comment:
Non-blocking: I wonder if it would make more sense that these are `CollapsibleItemComponent` and the other styles etc have the `Collapsible` name?

Reply:
"Collapse" is the name of the Bootstrap component that powers all of this, so trying to stay consistent with that.